sdk/resourcemanager/README.md

```diff
@@ -80,7 +80,7 @@ In the case where you are interested in certain service above or the service not
 <dependency>
   <groupId>com.azure.resourcemanager</groupId>
   <artifactId>azure-resourcemanager-mediaservices</artifactId>
-  <version>2.3.0</version>
+  <version>2.4.0</version>
 </dependency>
 ```
 [//]: # ({x-version-update-end})
```

Comment:
nit, we probably should change the sample lib, as mediaservices get retired.

It does not need to be done in this PR.